### PR TITLE
fix_1959_tz

### PR DESCRIPTION
**Pull request by clanmills**
I don't know why test_issue_1959.py was removed in b318b9d.  I'm restoring it, adding to test_reference_files and evading the TZ issue discussed in #2083.

@postscript-dev 1959 was provided by IPTC in #1959.  I don't think they will object to this change, however it would be courteous to let the user know that their file has been modified.

### CODE REVIEW

File: test/data/issue_1959_poc.xmp [hunk 1]
```diff
@@ -33,7 +33,6 @@
    exif:ExifVersion="0232"
    exif:FlashpixVersion="0100"
    exif:ColorSpace="65535"
-   photoshop:DateCreated="2022-01-04T09:41:01+00:00"
```
Comment (piponazo) [inline test/data/issue_1959_poc.xmp:36]: Interesting, you took the same approach I wanted to took here: #2079.

For me it is fine to recover the test case like this.

**Review by piponazo — APPROVED** [review]

File: test/data/issue_1959_poc.xmp (hunk 1 above)
Comment (clanmills) [reply test/data/issue_1959_poc.xmp:36]: Great minds think alike!

it's not really a fix.  It's evasion.  It's the metadata convertors.  I don't know who thought of adding convertors to the code base.  If those convertors are not defined in a standard, they should be implemented in another different project which might use libexiv2.  However, I don't know why this is in the metadata engine.

**Comment by hassec** [comment]
The removal of this `test_issue_1959.py` was discussed and agreed on in #2091 
There is no need for that test, and it would only duplicate the output reference file.

**Review by hassec — CHANGES_REQUESTED** [review]
The removal of `test_issue_1959.py` was discussed and agreed on in #2091, based on that it would only duplicate the output reference file.

`test/data/issue_1959_poc.xmp.out` is now basically a somewhat less verbose copy of `test/data/test_reference_files/issue_1959_poc.xmp.out`

That said, I don't have a strong opinion and am happy to go with whatever the crew decides.  
But what would be the benefit of adding this test twice?

**Comment by clanmills** [comment]
Ah, right.  I'm happy to drop this.  I wrongly thought this was "man overboard".

About the TZ issue with that test.  I assume Luis's @piponazo fix #2053 covers with that.
